support-frontend/assets/helpers/tracking/thirdPartyTrackingConsent.js

```diff
@@ -13,16 +16,48 @@ export type ThirdPartyTrackingConsent = typeof OptedIn
   | typeof OptedOut
   | typeof Unset;
 
-const getTrackingConsent = (): ThirdPartyTrackingConsent => {
+const getTrackingConsent = (): Promise<ThirdPartyTrackingConsent> => {
+  if (ccpaEnabled()) {
+    return new Promise((resolve) => {
+      onIabConsentNotification((consentState: boolean) => {
+        /**
+         * In CCPA mode consentState will be a boolean.
+         * In non-CCPA mode consentState will be an Object.
+         * Check whether consentState is valid (a boolean).
+         * */
+        if (typeof consentState !== 'boolean') {
+          throw new Error('consentState not a boolean');
+        } else {
+          // consentState true means the user has OptedOut
+          resolve(consentState ? OptedOut : OptedIn);
+        }
+      });
+    }).catch((err) => {
+      console.log('***** ERR *****', err);
```

Comment:
Was leaving this in deliberate or just for testing purposes?

Reply:
thanks for pointing out, that should be removed